src/main/java/org/mongoflink/sink/MongoCommitter.java

```diff
@@ -38,20 +38,30 @@ public class MongoCommitter implements Committer<DocumentBulk> {
             .writeConcern(WriteConcern.MAJORITY)
             .build();
 
+    private final boolean enableUpsert;
+    private final String[] upsertKeys;
+
     private final static long TRANSACTION_TIMEOUT_MS = 60_000L;
 
-    public MongoCommitter(MongoClientProvider clientProvider) {
+    public MongoCommitter(MongoClientProvider clientProvider, boolean enableUpsert, String[] upsertKeys) {
```

Comment:
Could we keep the old method and set the default value for the new parameters?

For example, 
```
public MongoCommitter(MongoClientProvider clientProvider) {
    this(clientProvider, false, new String[]{});
}
```

Reply:
ok.